lib/Gitlab/Client.php

```diff
@@ -54,48 +58,62 @@ class Client
     const AUTH_OAUTH_TOKEN = 'oauth_token';
 
     /**
-     * @var array
+     * @var string
      */
-    private $options = array(
-        'user_agent'  => 'php-gitlab-api (http://github.com/m4tthumphrey/php-gitlab-api)',
-        'timeout'     => 60
-    );
+    private $userAgent = 'php-gitlab-api (http://github.com/m4tthumphrey/php-gitlab-api)';
```

Comment:
Users where able to read it through `getOption('user_agent')`. I added it to limit the breaking changes surface.